apps/web-giddh/src/app/vouchers/list/list.component.ts

```
@@ -334,6 +382,70 @@ export class VoucherListComponent implements OnInit, OnDestroy {
+        this.settingForm.get('invoiceSettings.gstEInvoiceEnable')?.valueChanges.pipe(
+            debounceTime(700),
+            distinctUntilChanged(),
+            takeUntil(this.destroyed$),
+        ).subscribe(value => {
+            if (value !== null && value !== undefined) {
+                if (this.settingForm.get('invoiceSettings.gstEInvoiceEnable').value) {
+                    return;
+                } else {
+                    this.settingForm.get('invoiceSettings.generateEinvoiceShowPopUp').patchValue(false);
+                }
+                if (this.activeCompany?.subscription?.planDetails?.restrictedModules.hasOwnProperty(
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)